ipinfo/getip.go

```
@@ -0,0 +1,152 @@
+import (
+	"encoding/json"
+	"regexp"
+	"speedtest/config"
+	"speedtest/results"
+	"strings"
+
+	"github.com/WJQSERVER-STUDIO/go-utils/logger"
+)
+
+var (
+	logw       = logger.Logw
```

golangci-lint (1.62.2)

var `logw` is unused

(unused)